react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,15 @@
+.fa-times {
+  color: red;
+}
```

Comment:
It is not a good idea to directly change the behavior of a class from a CSS library. This can cause confusion with other people working on a project or even cause confusion for you in other parts of the project. A better approach is to make a class that does what you need it to do then use in in conjunction with the css library class. For example here you could make a class named something like `.text-red`